---
title: "Metagenomic Microbiome Disruption Indices"
subtitle: "<hr>"
author: "Brendan J. Kelly, MD, MS"
institute: "Infectious Diseases, Epidemiology, & Microbiology" 
date: "`r format(Sys.Date(), '%d %B %Y')`"
output:
  xaringan::moon_reader:
    css: xaringan-themer-inferno-grids-fade.css
    lib_dir: libs
    chakra: libs/remark-latest.min.js
    seal: false
    nature:
      ratio: 16:9
      highlightStyle: github
      highlightLines: true
      countIncrementalSlides: false
      slideNumberFormat: "%current%"

---
exclude: true

```{r setup, include=FALSE, echo=FALSE, message=FALSE, error=FALSE, include=FALSE, comment=FALSE}
options(htmltools.dir.version = FALSE)
xaringan::summon_remark()
library(tidyverse)
library(gt)

```

```{r xaringan-tile-view, echo=FALSE, message=FALSE, error=FALSE, include=FALSE, comment=FALSE}
xaringanExtra::use_tile_view()
xaringanExtra::use_webcam()
xaringanExtra::use_tachyons()
xaringanExtra::use_animate_css()
xaringanExtra::use_share_again()
#xaringanExtra::use_scribble(pen_color = "#56106E")
```

---
class: top left hide-count
background-color: #00688b
background-size: cover

.move-top10[
.title-text[<span style='font-size:1.1em;color:#202020;text-shadow: #FFFFFF 1px 0 10px;'>Metagenomic Microbiome Disruption Indices</span>]  
.title-subtext[<span style='font-size:0.8em'></span>]  
]  
  
.callout-url-leftcorner[
.title-nametext[
Brendan J. Kelly, MD, MS  
Infectious Diseases, Epidemiology & Microbiology  
University of Pennsylvania  
`r format(Sys.Date(), '%d %B %Y')`   
]
]

<!-- .footnote-right[<span style='color: white'>photo: Daniel Norris @unsplash</span>]   -->



---
exclude: true

## Disclosures  

.pad-left[  

- No conflicts of interest.  

- Research supported by:

    - NIAID K23 AI121485  
    - CDC BAA 200-2016-91964  
    - CDC BAA 200-2016-91937  
    - CDC BAA 200-2018-02919  
    - CDC BAA 200-2021-10986   
    - CDC Prevention Epicenters U54CK000610  
    - NIAID U19AI174998  

]

---
class: top left

.pull-left[

### Sequence-Based Diagnostics

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='85%'}
knitr::include_graphics(path = "img/karius_example_report.png")

```

- plasma cell-free DNA (e.g., Karius)  
  
- sequencing to diagnose prosthetic joint infection, CSF metagenomics, etc  

]

.pull-right[

### Live Biotherapeutic Products    

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='85%'}
knitr::include_graphics(path = "img/ve303_kmplot.png")

```

- 2 first-generation LBPs FDA-approved  
  
- 1 second-generation LBP in phase 3  

]

.footnote-left[Blauwkamp TA et al _Nature Micro_ 2023; Ivy MI et al _J Clin Micro_ 2018; Wilson MR et al _NEJM_ 2019; Khanna S et al _Drugs_ 2022; Feuerstadt P et al _NEJM_ 2022; Louie T et al _JAMA_ 2023]  


---

## Outline  

.pad-left[
- Microbiome disruption indices (MDIs) for _diagnosis_:

    - bacterial community diversity (e.g., Shannon diversity)  

    - bacterial community dominance  

    - absolute bacterial abundance  

    - combinations of above  

- In addition to diagnostic utility, role in quantifying _colonization resistance_: 

    - features of resident mucosal microbiota that resist introduction of new species (including pathogens) via direct competition and/or host immune modulation

]


.footnote-left[Blaser _Science_ 2016; Raymond et al _ISME J_ 2016; Kelly & Tebas _Chest_ 2018; Al-Hasan et al _J Infect_ 2010; Jensen et al _CMI_ 2011; Munoz et al _CMI_ 2016]  


---

## MDI Questions         

.pad-left[

- Diagnostic vs predictive intent?  

- Longitudinal vs cross-sectional?  

- Proportional vs absolute abundance?  

- Single site vs comparative?  

]


---

## Microbiome Disruption Indices  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='55%'}
knitr::include_graphics(path = "figs/harrigan_cid_2021_oddsratio.png")

```

.footnote-left[Harrigan et al _CID_ 2021]  


---

## Microbiome Disruption Indices  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='60%'}
knitr::include_graphics(path = "figs/p_toxineia_clostridioides_boxplot_tp.png")

```


.footnote-left[Kevin Alby, Colin Tkatch, John Jachimiak. Submitted.]  


---

## Antibiotics & Microbiome Disruption  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='50%'}
knitr::include_graphics(path = "figs/ziegler_ofid_2019_pcoa.png")

```


.footnote-left[Ziegler et al _OFID_ 2019]  



---

## Microbiome Disruption Indices  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='59%'}
knitr::include_graphics(path = "img/bittinger_charlson_enrichment.png")

```

.footnote-left[Charlson ES et al _AJRCCM_ 2012]  


---

## Urine Metagenomics: Wilkins & Anesi Study  

.pad-left[
-  Kidney transplantation (KT): prior studies have documented changes in the intestinal microbiome composition and function (dysbiosis) following KT. 

- Question of impact on infectious and inflammatory (e.g., rejection) outcomes. 

- Pilot feasibility study: the urinary microbiome may have even more direct impact on post-KT outcomes.

- Is metagenomic urine microbiome profiling feasible and informative in post-KT population?   
]


---

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='75%'}
library(tidyverse)
library(gt)
library(gtsummary)

t1 <- read_rds("data/table1_urineindication_exposure.rds")

t1

```


---

## Urine Metagenomics: Wilkins & Anesi Study  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='55%'}
library(tidyverse)

# p_read_taxon_percentage_color_by_ncbi <- read_rds("data/p_read_taxon_percentage_color_by_ncbi.rds")

# p1 <- p_read_taxon_percentage_color_by_ncbi + scale_y_discrete(limits = rev) + theme(legend.title = ggtext::element_markdown()) + labs(x = "Proportion of Metagenomic<br>Reads Assigned", fill = "NCBI<br>Taxonomy") + guides(fill=guide_legend(nrow=3, byrow=TRUE))
# p1

knitr::include_graphics(path = "figs/p_read_taxon_percentage_color_by_ncbi.png")


```


---

## Urine Metagenomics: Wilkins & Anesi Study  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='55%'}
library(tidyverse)

# p_read_species_percent_bacterial_by_indication <- read_rds("data/p_read_species_percent_bacterial_by_indication.rds")

# p2 <- p_read_species_percent_bacterial_by_indication + labs(y = "Proportion of Reads Assigned to Bacteria")
# p2

knitr::include_graphics(path = "figs/p_read_species_percent_bacterial_by_indication.png")


```


---

## Urine Metagenomics: Wilkins & Anesi Study  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='65%'}
library(tidyverse)

# p_read_species_percent_bacterial_by_HStime_facet_symptoms <- read_rds("data/p_read_species_percent_bacterial_by_HStime_facet_symptoms.rds")

# p_read_species_percent_bacterial_by_HStime_facet_symptoms

knitr::include_graphics(path = "figs/p_read_species_percent_bacterial_by_HStime_facet_symptoms.png")


```


---

## Urine Metagenomics: Wilkins & Anesi Study  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='95%'}
library(tidyverse)

# p_resist_kappa_fixed_pred_simple <- read_rds("data/p_resist_kappa_fixed_pred_simple.rds")

# p_resist_kappa_fixed_pred_simple

knitr::include_graphics(path = "figs/p_resist_kappa_fixed_pred_simple.png")


```


---

## Urine Metagenomics: Host vs Bacterial Reads  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='65%'}
library(tidyverse)

knitr::include_graphics(path = "figs/seq_tool_comp_domain_plot_sra_sc.png")


```


---

## Urine Metagenomics: Host vs Bacterial Reads  

```{r, echo=FALSE, echo=FALSE, message=FALSE, warning=FALSE, error=FALSE, fig.align='center', out.width='65%'}
library(tidyverse)

knitr::include_graphics(path = "figs/seq_tool_comp_domain_plot_sra_tc.png")


```


---

## Future Directions      
  
.pad-left[

- MDIs: 

    - diagnostic utility

    - markers of colonization resistance  

    - risk for MDRO colonization/infection  

- Approaches:  

    - amplicon vs metagenomic: value of host signal?  

    - integration of metagenomic pipelines?  
  
]


---
exclude: true

## Acknowledgements    

.pull-left[
- __MOTION Lab & C-STAR @ Penn__
Laura Cowden, Laurel Glaser, Kyle Rodino, Sean Loughrey, 
Magda Wernovsky, Erik Clarke, Michael David, Matt Ziegler, 
Lauren Dutcher, Ebbing Lautenbach, Jim Harrigan, Alexa Patel, Dylan Tapper, Anne Jaskowiak, Pam Tolomeo   

- __Bushman Laboratory @ Penn__
Arwa Abbas, Aoife Roche, Andrew Marques,
Aubrey Bailey, John Everett,
Rick Bushman

- __PennCHOP Microbiome Program__
Lisa Mattei, Huanjia Zhang, Kyle Bittinger

]


.pull-right[
- __Collman Laboratory @ Penn__
Ize Imai, Aurea Simon Soro, John McGinniss, Ron Collman

- __Division of ID @ Penn__
Ian Frank, Pablo Tebas, Robert Gross, Emily Blumberg

- __Rutgers University & Penn DBEI__
Jason Roy, Arman Oganisian
  
- __CDC Prevention Epicenters__
Clifford McDonald, Alison Laufer Halpin
  
- __Funding__
<u>CDC</u>: BAAs 200-2016-91964, 200-2018-02919, 200-2021-10986,
200-2021-10986 & <u>NIAID</u>: K23 AI121485

]

.center[
### brendank@pennmedicine.upenn.edu
]



---
exclude: true

class: middle center hide-count
background-image: url(img/giorgio-trovato-IclGoWB6wEY-unsplash.jpg)
background-size: cover

.title-subtext[Questions?]



---
exclude: true
class: middle center hide-count
background-image: url(img/giorgio-trovato-IclGoWB6wEY-unsplash.jpg)
background-size: cover

.title-subtext[Questions?]

.callout-url-bottom[  
.fade-in[<span style="font-size:0.5em; color:white">slides â†“ </span>]  
<span style="link-color:white">[bjklab.org](http://www.bjklab.org)</span>  
]  


---
class: top left hide-count
background-color: #00688b
background-size: cover


